src/ServiceProviders/RevenueServiceProvider.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace  Give\ServiceProviders;
+
+use Give\Revenue\Repositories\Revenue as RevenueRepository;
+use Give\Revenue\Database\Revenue;
+
+/**
+ * Class DatabaseTables
+ * @package Give\ServiceProviders
+ *
+ * @since 2.9.0
+ */
+class RevenueServiceProvider implements ServiceProvider {
```

Comment:
@ravinderk Please move this class to `src/Revenue` — ultimately we'll get rid of the `src/ServiceProviders` directory.

Reply:
fixed: https://github.com/impress-org/givewp/compare/ccc086f6f8a9...9e31b223eb33